AD419.DataHelper.Web/Controllers/FFYExpensesByARCReportController.cs

```diff
@@ -28,20 +25,30 @@ public ActionResult Index()
                         Account = a.Account,
                         Comments = a.Comments
 
-                    }).ToList();            
+                    }).ToList();
+
+            model.UseStateFiscalYear = false;
 
             return View(model);
         }
 
-        public ActionResult Report()
+        public ActionResult Report(int id)
         {
-            var reportName = @"Direct and Indirect FFY Expenses by ARC w Account";
+            var useStateFiscalYear = id;
+            var fiscalYearTitleSegment = useStateFiscalYear == 0 ? "FFY" : "SFY";

```

Comment:
@taylorkj Definitely don't like this and i bet you don't either -- could make it a bool instead of an int and then we'd have a true false instead of where == 0.  And of course not naming it id would be better too, Report(bool useFiscalYear) makes more sense.